### PR TITLE
updated the onboarding steps for po and vendor

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**
1. updated the onboarding steps for po and vendor
2. Included the video to set up MFA for WOG account in PO onboarding.
The video was reviewed by Jerome.

### CODE REVIEW

File: docs/onboard-vendors-to-techpass.md [hunk 1]
```diff
@@ -34,43 +33,42 @@ Vendors or contractors who do not have a non-SE GSIB device
 
 1. Contact your project manager or reporting officer to request for TechPass account and SEED provisioning.
 
-2. Provide the required details in this request such as your official vendor company email address, mobile phone number and project name.
+2. Provide the required details in this request such as your organisational email address provided by your vendor organisation, mobile phone number and project name.
```
Comment (yilunyou) [inline docs/onboard-vendors-to-techpass.md]:
```suggestion
2. Provide the required details in this request such as your organisation email address provided by your vendor organisation, mobile phone number and project name.
```
probably shouldn't use an adjective?

File: docs/onboard-to-techpass.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Onboard to TechPass
+
+Users can onboard to TechPass as **public officers** or **vendors**.
+
+Refer to the following table to identify your onboarding persona:
+
+| Persona| Description | <div style="width:210px">Examples</div> |
+|----| ------------- |:-------------:|
+| **Vendor** | Users who do not have a WOG account. These users may have an email address provided by the vendor organisation or it may belong to specific domains such as<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- dsta.gov.sg<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- dsta-wog.gov.sg<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- mindef.gov.sg<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- defence.gov.sg<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- gebiz.gov.sg<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- sps.gov.sg<br><br>**Note**: Email domain is the part of an email address that comes after the “@” symbol. For example, if your email address is john_doe@sps.gov.sg, then **sps.gov.sg** is your email domain.| - john_doe@ncs.com.sg<br>- john_doe@accenture.com.sg<br>- john_doe@dsta.gov.sg<br>- john_doe@gebiz.gov.sg  |
```
Comment (yilunyou) [inline docs/onboard-to-techpass.md]: sps.gov.sg
is this list from JunJie?

File: docs/onboard-vendors-to-techpass.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply docs/onboard-vendors-to-techpass.md]: Yilun, I wanted to be specific that this email address is their official email address, and Microsoft uses Organisational account.

image

File: docs/onboard-to-techpass.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply docs/onboard-to-techpass.md]: Updated. I have included a statement that vendors can't use their personal email id.

**Review by yilunyou — APPROVED** [review]

**Comment by SenthilKumarSivakami** [comment]
Some of the changes done here is outdated now. so closing this PR.

**Pull request closed without merging**